app/src/main/res/values/strings.xml

```diff
@@ -6,13 +6,17 @@
     <string name="entry_status_line">%1$s (%2$d/%3$d)</string>
     <string name="database_name_testing">test-gather_words</string>
     <string name="database_name">gather_words</string>
+    <string name="Undo">undo</string>
 
     <string-array name="word_lists">
         <item>None (Blank Template)</item>
         <item>swadesh-100</item>
         <item>swadesh-207</item>
     </string-array>
 
+    <!--MainActivity strings-->
```

Comment:
Category already exists on line 28.

Reply:
Do you know any good eye doctors?

Fixed.